### PR TITLE
Add schedule timezone on trigger page

### PR DESCRIPTION
**Pull request by prateek2211**
fixes moira-alert/moira#472
Screenshot from 2020-01-29 17-19-48

### CODE REVIEW

**Comment by beevee** [comment]
Wow, that's a lot of linter errors for such a small PR :-)

File: src/Components/TriggerInfo/TriggerInfo.jsx [hunk 1]
```diff
@@ -61,9 +61,14 @@ function ScheduleView(props: { data: Schedule }): React.Node {
         .format("HH:mm")}–${moment("1900-01-01 00:00:00")
         .add(endOffset, "minutes")
         .format("HH:mm")}`;
+
+    const timeZone = "GMT " + (tzOffset < 0 ? "- " : "+ ") +
+        `0${Math.abs(tzOffset) / 60 ^ 0}`.slice(-2) + ':' +
+        ('0' + Math.abs(tzOffset) % 60).slice(-2);
```
Comment (litleleprikon) [inline src/Components/TriggerInfo/TriggerInfo.jsx]: In my opinion this variable declaration is overloaded and not clear enough. Let's divide it to different operations like calculation of hours, minutes and definition of plus or minus sign

**Comment by sashasushko** [comment]
@prateek2211 You can use `yarn lint` command to get list of errors or try use autofix. Or (the best way) add eslint autofix to precommit hook 😉

File: src/Components/TriggerInfo/TriggerInfo.jsx (hunk 1 above)
Comment (sashasushko) [reply src/Components/TriggerInfo/TriggerInfo.jsx]: Also we already have moment.js in dependencies. @prateek2211 You can try to use it to get GMT

**Comment by sashasushko** [comment]
@prateek2211 thank you. I put some fixes in 50cd47ba2a635679c07578c3269850489fed43e7, but all stars to you

**Pull request merged**